From 433390ba4eedf8aae3149957b61c068c627203cb Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:28 -0400
Subject: [misc] mei: me: put pm callbacks under PM_SLEEP ifdef

Message-id: <1399041470-21330-56-git-send-email-prarit@redhat.com>
Patchwork-id: 79342
O-Subject: [RHEL7.1 PATCH BZ 1088953 55/77] mei: me: put pm callbacks under PM_SLEEP ifdef
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 16833257a1ee9f7b2c997f0a26f91b1d9055642c
Author: Alexander Usyskin <alexander.usyskin@intel.com>
Date:   Tue Feb 18 14:31:08 2014 +0200

    mei: me: put pm callbacks under PM_SLEEP ifdef

    This will eliminate compilation warning:

    drivers/misc/mei/pci-me.c:303:12: warning: mei_me_pci_suspend defined but not used [-Wunused-function]
    static int mei_me_pci_suspend(struct device *device)
    drivers/misc/mei/pci-me.c:323:12: warning: mei_me_pci_resume defined but not used [-Wunused-function]
    static int mei_me_pci_resume(struct device *device)

    Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/pci-me.c b/drivers/misc/mei/pci-me.c
index 8b49be1..ffaf00e 100644
--- a/drivers/misc/mei/pci-me.c
+++ b/drivers/misc/mei/pci-me.c
@@ -305,7 +305,7 @@ static void mei_me_remove(struct pci_dev *pdev)
 
 
 }
-#ifdef CONFIG_PM
+#ifdef CONFIG_PM_SLEEP
 static int mei_me_pci_suspend(struct device *device)
 {
  struct pci_dev *pdev = to_pci_dev(device);
@@ -365,11 +365,12 @@ static int mei_me_pci_resume(struct device *device)
 
  return 0;
 }
+
 static SIMPLE_DEV_PM_OPS(mei_me_pm_ops, mei_me_pci_suspend, mei_me_pci_resume);
 #define MEI_ME_PM_OPS (&mei_me_pm_ops)
 #else
 #define MEI_ME_PM_OPS NULL
-#endif /* CONFIG_PM */
+#endif /* CONFIG_PM_SLEEP */
 /*
  *  PCI driver structure
  */
-- 
1.7.1